src/main/resources/animal-attribute-data.yml

```diff
@@ -64,34 +88,106 @@ animal-attribute-data:
       attributeName: SEX
       attributeValue: SHE
 
-    - animalType: REPTILE
-      attributeName: DANGEROUS
+    - animalType: FISH
+      attributeName: WATER
+      attributeValue: FRESH
+
+    - animalType: FISH
+      attributeName: WATER
+      attributeValue: SALT
+
+    - animalType: TURTLE
+      attributeName: SIZE
+      attributeValue: SMALL <0.5 KG
+
+    - animalType: TURTLE
+      attributeName: SIZE
+      attributeValue: MEDIUM 0.5-5 KG
+
+    - animalType: TURTLE
+      attributeName: SIZE
+      attributeValue: BIG >5 KG
+
+    - animalType: TURTLE
+      attributeName: AGE
+      attributeValue: YOUNG <2 YEARS
+
+    - animalType: TURTLE
+      attributeName: AGE
+      attributeValue: ADULT >2 YEARS
+
+    - animalType: SNAKE
+      attributeName: SIZE
+      attributeValue: SMALL <0.5 KG
+
+    - animalType: SNAKE
+      attributeName: SIZE
+      attributeValue: MEDIUM 0.5-2 KG
+
+    - animalType: SNAKE
+      attributeName: SIZE
+      attributeValue: BIG >2 KG
+
+    - animalType: SNAKE
+      attributeName: VENOMOUS
       attributeValue: "YES"
 
-    - animalType: REPTILE
-      attributeName: DANGEROUS
+    - animalType: SNAKE
+      attributeName: VENOMOUS
       attributeValue: "NO"
 
-    - animalType: REPTILE
-      attributeName: SEX
-      attributeValue: MALE
+    - animalType: HAMSTER
+      attributeName: SIZE
+      attributeValue: SMALL <0.05 KG
 
-    - animalType: REPTILE
-      attributeName: SEX
-      attributeValue: SHE
+    - animalType: HAMSTER
+      attributeName: SIZE
+      attributeValue: MEDIUM 0.05-0.15 KG
 
-    - animalType: HORSE
+    - animalType: HAMSTER
       attributeName: SIZE
-      attributeValue: BIG
+      attributeValue: BIG >0.15 KG
 
```

Comment:
Czy rozmiar ma znaczenie? :grin:

Reply:
no wlasnie tutaj nwm czy nie zrezygonwac z rozmiarow xD

Reply:
W przypadku chomikow, ptaków, żółwi itp to wywaliłbym te rozmiary bo pewnie mało kto wie ile waży jego ptaszek xD
Zostawiłbym dla psa, kota i może węża czy żółwia bo one mogą faktycznie trochę ważyć.